providers/azureblob/src/main/java/org/jclouds/azure/storage/filters/SharedKeyLiteAuthentication.java

```diff
@@ -216,6 +271,7 @@ public String calculateSignature(String toSign) throws HttpException {
       String signature = signString(toSign);
       if (signatureWire.enabled())
          signatureWire.input(Strings2.toInputStream(signature));
+      System.out.println(signature);
```

Comment:
Change this for logs if you really need this output, otherwise let's just remove this.

Reply:
Yes, this was just a debugging aid, I forgot to remove it.